examples/north_sea/model_config.py

```diff
@@ -74,20 +76,18 @@ def construct_solver(spinup=False, store_station_time_series=True, **model_optio
         simulation and a function for updating forcings
     """
 
+    pwd = os.path.abspath(os.path.dirname(__file__))
+    h5_file_name = os.path.join(pwd, "north_sea_bathymetry.h5")
+    with CheckpointFile(h5_file_name, "r") as f:
+        mesh2d = f.load_mesh("firedrake_default")
+        bathymetry_2d = f.load_function(mesh2d, "Bathymetry")
```

Comment:
Please could you also make this and other changes related to setting up the bathymetry in `demos/demo_2d_north_sea.py`?

Reply:
Good point. Done